deploy/kustomize/README.md

```diff
@@ -0,0 +1,44 @@
+# Intro
```

Comment:
I recommend making the header something like "Kustomize Manifests" and simply describe this directory as manifests for delivering and deploying the HCO via OLM using Kustomize.

Reply:
Instead of having the "Deploy Modes" as you do below. I recommend that here, you mention that there are two necessary steps to install HCO via OLM in a cluster 1) "delivery" - make OLM aware of HCO and 2) "deploy" - use OLM provided APIs to install HCO.